lib/measured/conversion.rb

```diff
@@ -1,20 +1,14 @@
 class Measured::Conversion
   ARBITRARY_CONVERSION_PRECISION = 20
-  def initialize(case_sensitive: false)
-    @base_unit = nil
-    @units = []
-    @case_sensitive = case_sensitive
-  end
-
-  attr_reader :base_unit, :units, :case_sensitive
 
-  def set_base(unit_name, aliases: [])
-    add_new_unit(unit_name, aliases: aliases, base: true)
+  def initialize(base_unit, units, case_sensitive: false)
+    @case_sensitive = case_sensitive
+    @base_unit = base_unit
+    @units = units.dup
+    @units << @base_unit
```

Comment:
This simplifies the interface to `Conversion`. I can `@units.uniq!` too, just to be sure.